examples/calib_pulses.py

```diff
@@ -0,0 +1,98 @@
+r"""
+Calibrate pulses output from RF amplifier
+=========================================
+If calibrating the pulse lengths, a series of pulse lengths are sent to the 
+spincore directly and the output pulse is captured on the GDS oscilloscope.
+If testing the calibration or capturing using a series of desired betas,
+the calibrating conditional should be set to false and the script will calibrate
+the pulse lengths so that the output of the amplifier produces the desired beta.
+It is very important to note that there MUST BE at least a 40 dBm
+attenuator between the RF amplifier output and the GDS oscilloscope input to
+avoid damaging the instrumentation! It is advised that the attenuator be
+calibrated using the GDS and AFG beforehand
+"""
+import pyspecdata as psd
+import os
+import SpinCore_pp as spc
+from datetime import datetime
+from Instruments import GDS_scope
+import numpy as np
+
+calibrating = True
+my_exp_type = "test_equipment"
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
+# {{{ importing acquisition parameters
+config_dict = spc.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    config_dict["SW_kHz"], config_dict["acq_time_ms"]
+)
+# }}}
+# {{{ add file saving parameters to config dict
+config_dict["type"] = "pulse_calib"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["misc_counter"] += 1
+# }}}
+# {{{ ppg
+sqrt_P = config_dict["amplitude"] * np.sqrt(75)  # we have a 75 W amplifier
+desired_beta = np.linspace(0.5 * sqrt_P, 100, 50)
```

Comment:
if you are calibrating, you are not calculating desired beta.  Also, desired beta should not change with the amplitude.  Which means we are not interested in this sqrt_P.